http-clients/netty-nio-client/src/main/java/software/amazon/awssdk/http/nio/netty/internal/ChannelPipelineInitializer.java

```diff
@@ -94,6 +96,8 @@ public void channelCreated(Channel ch) {
 
             pipeline.addLast(sslHandler);
             pipeline.addLast(SslCloseCompletionEventHandler.getInstance());
+
```

Comment:
Can we add a comment here to explain why we are setting this?
Probably something like this: 

`// Use unpooled allocator to avoid increased heap memory usage from Netty 4.1.43. See https://github.com/netty/netty/issues/9768`